### PR TITLE
[FIXED JENKINS-31192] Fix HTTPS SSL client certificate authentication

### PR DESCRIPTION
**Pull request by ymartin59**
[JENKINS-31192](https://issues.jenkins-ci.org/browse/JENKINS-31192)

Take benefits of new SVNKit 1.8.11 SVNSSLAuthentication methods to properly support byte array PKCS#12 certificate usage.

Supersede https://github.com/jenkinsci/subversion-plugin/pull/141

### CODE REVIEW

**Comment by ymartin59** [comment]
@recena Here it is. My job using client certificate authentication runs properly on my test environment. By the way, there is still a lot of job to switch from deprecated constructors to "newInstance" factory methods available in whole SVNAuthentication class hierarchy.

**Comment by recena** [comment]
@ymartin59 Looks great!

**Comment by recena** [comment]
@ymartin59 By the way, Which SVN Server are you using?

File: salida [hunk 1]
```diff
@@ -0,0 +1,261 @@
+[INFO] Scanning for projects...
```
Comment (recena) [inline salida]: @ymartin59 Please, remove this file. I uploaded my mistake.

**Comment by recena** [comment]
@ymartin59 Why did you close this PR?

**Comment by ymartin59** [comment]
I am "experimenting" GitHub. I have just delete the source branch to create a new one with same name. I have not expected GitHub to close PR but it sounds reasonable. Here is another one with cleaner history: #146

**Comment by recena** [comment]
@ymartin59 Please, try to recover this.

**Pull request closed without merging**